src/renderer/components/PopoverConfirm.tsx

```diff
@@ -29,12 +29,14 @@ export const PopoverConfirm: FC<PopoverConfirmProps> = ({
   // cloneElement 保证传递事件，不破坏原子元素的所有 CSS
   let target = children
   if (isValidElement(children)) {
-    target = cloneElement(children, {
-      onClick: (e) => {
-        if (children.props.onClick) children.props.onClick(e)
-        setOpened(true)
+    const elementWithOnClick = children as React.ReactElement<{ onClick?: (e: React.MouseEvent) => void }>;
+
+    target = cloneElement(elementWithOnClick, {
+      onClick: (e: React.MouseEvent) => {
+        if (elementWithOnClick.props.onClick) elementWithOnClick.props.onClick(e);
+        setOpened(true);
```

Comment:
I think this is better
```suggestion
 if (isValidElement<{ onClick?: (e: React.MouseEvent) => void }>(children)) {
    target = cloneElement(children as ReactElement<{ onClick?: (e: React.MouseEvent) => void }>, {
      onClick: (e: React.MouseEvent) => {
        if (typeof children.props.onClick === 'function') {
          children.props.onClick(e)
        }
        setOpened(true)
```